README.md

```diff
@@ -12,53 +12,144 @@ A resource may classify its messages into groupings,
 and it may include additional data or metadata relating to them.
 It should be possible to represent a resource in a text-based human-friendly manner.
 
-### How to Contribute
-
-- **Issues** are for specific aspects of the resulting specification.
-  Discussions here should be focused, and stay on topic.
-  New issues must include a sufficient initial description to introduce themselves,
-  and may contain an exploration of possible solutions to them.
-- **Pull requests** help develop and define the eventual specification.
-  Accepting and merging a PR should not be taken as the final word on any topic,
-  but each change should be an improvement on the previous.
-  Each PR should only apply one change that may be squashed to the `main` branch.
-  Text content should use [semantic line breaks](https://sembr.org/).
-  A PR may close one or more issues, but this is not required.
-- **Discussions** are freeform, and may well refer to multiple issues or topics.
-  This is also the forum in which we should develop further our ways of working,
-  which may also include interactions and forums outside this repository,
-  such as occasional video calls.
-
-We welcome your participation and interest!
-
-## Syntax
-
-As currently specified, a message resource looks like this (syntax highlighting only approximate):
+This work was [presented](https://www.youtube.com/watch?v=ksgm_B-uUCU)
+at the [2024 Unicode Tech Workshop](https://www.unicode.org/events/utw/2024/).
+
+### Why?
+
+The work on MessageFormat 2 has identified the following challenges
+that go beyond or arise from the new message format,
+but which are not well addressed by existing resource formats:
+
+- The MessageFormat 2 syntax is naturally multi-line,
+  and multi-line values are not easy to use in many of the current resource formats.
+- Comments and metadata are important for translators,
+  but (mostly) not relevant for formatting.
+  Their attachment to messages needs to be well specified,
```

Comment:
As far as I understand comments can be bound to groupings (sections). If that's correct, worth calling it out?

Possible alternatives to attachment: connection, link, binding? Later on you use "assign to", I would keep the terminology consistent.

Reply:
> As far as I understand comments can be bound to groupings (sections). If that's correct, worth calling it out?

That is a feature of the strawman proposal, yes, but I don't think it's necessarily a must-have part of the solution. This section is about presenting the problem.

> Possible alternatives to attachment: connection, link, binding? Later on you use "assign to", I would keep the terminology consistent.

Agreed about consistency, "attachment" is I think the best term here.